src/components/views/investors/Login.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import Field from '../../common/Fields.jsx';
+
+
+const fields = [
+  {
+    name: 'email', type: 'email', placeholder: 'Please enter your email address', label: 'Email Address'
+  },
+  {
+    name: 'password', type: 'password', placeholder: 'Please enter your password', label: 'Password'
+  }
+];
+
+class Login extends Component {
+  render() {
+    return (
+        <div className="login-wrapper">
+          <form action="">
+            <div className='login-form-header'>
+              <h1>{this.props.header}</h1>
+              <h2>{this.props.subHeader}</h2>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['subHeader' is missing in props validation](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)